tests/api/test_rendering.py

```diff
@@ -118,3 +124,25 @@ def test_contex_can_access_environment_variable(cli_context):
     assert snowflake_sql_jinja_render("&{ ctx.env.TEST_ENV_VAR }") == os.environ.get(
         "TEST_ENV_VAR"
     )
+
+
+def test_has_sql_templates():
+    assert has_sql_templates("abc <% %> abc")
+    assert has_sql_templates("abc <% abc")
```

Comment:
This example does not have SQL template, right?

Reply:
From Jinja perspective it's considered it has it, and it will fail when expanding.